daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -232,10 +232,10 @@ function Change-InstanceProperties {
   # Change time zone to Coordinated Universal Time.
   Run-Command tzutil /s 'UTC'
 
-  # Not supported on 6.1 client, but is supported on 6.1 server
+  # Supported on 6.1 client and 6.1 server with SP1 installed only
```

Comment:
Hibernate is only supported on Windows 8+ clients and Windows 2008R2 SP1+ servers.